source/language/types.rst

```diff
@@ -268,6 +268,72 @@ are all of type ``float[64]``.
 
 Note that `e` is a valid identifier. `e/E` are also used in scientific notation where appropriate.
 
+Mathematical functions available for constant initialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to simple arithmetic functions used in expressions initializing constants,
+OpenQASM 3 offers the following built-in mathematical operators followed by
+their argument expression in parentheses:
+
+.. container::
+   :name: tab:built-in-math
+
+   .. table:: Built-in mathematical functions in OpenQASM3.
+
+      +----------+--------------------------------+---------------------+
+      | Function | [Input Range/Type] , [...]     | [Output Range/Type] |
+      +==========+================================+=====================+
+      | arccos   | [-1, 1]                        | [0, pi rad]         |
+      +----------+--------------------------------+---------------------+
+      | arcsin   | [-1, 1]                        | [-pi/2, pi/2 rad]   |
```

Comment:
We resolved in a different comment to put the type information in for every function, and specify the ranges where appropriate.  So perhaps these two should be `[float: [-1, 1]]`, or put the ranges in a separate "notes" column?